s27816-MichaelWajda/zajecia1/instalacja_debian.txt

```diff
@@ -0,0 +1,454 @@
+błędy w wyświetlaniu, które mogą powodować polskie znaki.
+22) Keymap to use
+American English - j.w. + klawiatura ze znakami polskimi nie powszechna w 
+użytku.
+Następuje pobieranie podstawowych komponentów systemu pod dalszą instalację 
+oraz przydzielenie adresu sieciowego. 
+23) Hostname - Debian12-demo
+Aby przełączyć się pomiędzy opcjami używamy przycisku Tab, Enter - 
+wybranie, Esc - powrót do menu z krokami instalacji, przyciski strzałek 
+przejdą z powrotem do pola tekstowego bądź obecnego wyboru.
+Po ustawieniu nazwy host'a zostaniemy zapytani o nazwę domeny, której 
+użyjemy, dla Pjatk'u jest to pjwstk.edu.pl natomiast w przypadku sieci 
+domowej, można wymyśleć nazwę domeny, ale należy pamiętać, 
+żeby używać tej samej nazwy na wszystkich maszynach. Można pominąć ten 
+krok nic nie wpisując i przechodząc dalej.
+24) Set up users and Passwords
+Tutaj należy skonfigurować hasło administratora/'root'. Warto zaznaczyć, 
+że nie należy pracować na użytkowniku root/administratorze, 
+gdyż w przypadku przejęcia konta, nie będzie na nim żadnych ograniczeń 
+dostępu do plików systemowych czy danych znajdujących się na maszynie. 
+Tworzę więc nowego użytkownika i nadaję mu hasło.
+Podział na partycje
+25) Partitioning Method
+Wybieramy opcję 'Guided Partitioning' dla tej instalacji, żeby dowiedzieć 
+się jakie partycje należy stworzyć przy instalacji systemu.
+LVM - Linux Volume Manager, zmienia sposób zarządzania dyskami i pozwala na 
+dokonywanie operacji na partycjach bez konieczności restartu.
+26) Partition Disk
+Wybieramy dysk do partycjonowania, w tej instalacji mamy dostępny tylko jeden.
+SCSI3 (0,0,0) (sda) - 21.5GB ATA VBOX HARDDISK
+SCSI3 - Small Computer System Interface, kontroler
+(0,0,0) 
+Pierwsze Zero - Numer kontrolera SCSI. Kontrolery SCSI mogą obsługiwać wiele 
+urządzeń SCSI i są numerowane od 0.
+Drugie Zero - Numer kanału (portu) na kontrolerze SCSI. Kanały również są 
+numerowane od 0.
+Trzecie Zero - Numer urządzenia na kanale SCSI. Urządzenia SCSI podłączone 
+do tego samego kontrolera i kanału są numerowane od 0.
+(sda) - identyfikator dla pierwszego dysku twardego, gdybyśmy podłączyli 
+kolejny dysk tego samego typu, nazywałby się już 'sdb'.
+21.5GB - mimo wcześniejszego wyboru 20GB ze względu na to, że używaną 
+przez Windows jednostką jest tak na prawdę Gibibajt (GiB - podział po 1024)
+ATA - interfejsu dysku, który w tym przypadku jest ATA (Advanced Technology 
+Attachment)
+VBOX - wskazuje, że dysk został utworzony w środowisku wirtualnym za pomocą 
+VirtualBox
+HARDDISK - rodzaj dysku, dysk twardy
+27) Następnie wybieramy opcję dzielącą dysk na partycje /home, /var i 
+/temp, co ułatwi nam diagnostykę błędów funkcjonowania systemu w razie ich 
+wystąpienia.
+/home (partycja w której znajdują się katalogi domowe dla wszystkich 
+użytkowników), 
+/var (variable - zmienne środowiskowe), 
+/temp (temporary - np. schowek)
+28) Tworzenie Partycje:
```

Comment:
Brak opisu flagi bootującej. Co to jest i do czego jest potrzebna. Kiedy powinna być ustawiona a kiedy nie.
Do poprawy.